contracts/upgradeable_contracts/native_to_erc20/ForeignBridgeNativeToErc.sol

```diff
@@ -34,6 +36,7 @@ contract ForeignBridgeNativeToErc is ERC677Receiver, BasicBridge, BasicForeignBr
         uintStorage[keccak256(abi.encodePacked("minPerTx"))] = _minPerTx;
         uintStorage[keccak256(abi.encodePacked("gasPrice"))] = _foreignGasPrice;
         uintStorage[keccak256(abi.encodePacked("requiredBlockConfirmations"))] = _requiredBlockConfirmations;
+        setOwner(_owner);
```

Comment:
Should `claimTokensFromErc677` be also allowed only for upgradability admin?

Reply:
Yes, also two other `claimTokens` overrides needed to be updated too. Fixed them on https://github.com/poanetwork/poa-bridge-contracts/pull/117/commits/adb81cf1ca663b668d4015c512287a0750323ff2

Reply:
Oh! Thanks!
And seams that we found a bug: the Home contact in `erc20-to-erc20` mode does not have 
```solidity
    function claimTokensFromErc677(address _token, address _to) external onlyProxyOwner {
        erc677token().claimTokens(_token, _to);
    }
```
similarly to the Foreign contract in `native-to-erc20` mode.

Reply:
Should we address this bug on a separate issue?

Reply:
Yes, please create an issue for this.

Reply:
Done. Created https://github.com/poanetwork/poa-bridge-contracts/issues/118